### ISSUE TITLE
Diastora Forest

### ISSUE THREAD

**Issue by one-forty** [NaNoGenMo/2018#15]
Recently I've been super into Dwarf Fortress and I am in awe of the complexity of the world/character generation, so I want to make a story where the characters are random (but not 100% random, still logical) and that affects what happens throughout the story.

The story will overall be about 4 men trying to get through a haunted forest, other than that I'm aiming for as many possibilities and outcomes as possible, all based  on the characters and their stats etc.

My main goal for this is, like DF, to have a seemingly infinite amount of stories to be generated from one program. When you're done reading, you will be able to press a button to generate a new story and read a new one.

**Comment by one-forty** [comment]
@LuRsT Thank you for being interested,

Yes all of them will be able to die but one will not until at least 50k lines to keep the story long enough, the deaths can happen at any point which will be great for suspense.

I didn't pay attention to the fact that it was only men but I will change that if it makes people happy, I may possibly add pets too.

In the forest they're just trying to survive, they have to get through it for some reason but I haven't figured that part out.

**Comment by cantino** [comment]
Cool idea! I am also intrigued by generative simulation (like DF).

**Comment by one-forty** [comment]
I added some skill points for each of the people, the values of each are randomly chosen at the start. You will not see the exact value, but have a description (eg. great strength, awful intelligence).

These skills are Strength, Toughness, Speed, Bravery, Focus and Intelligence

These will affect how each person reacts to certain situations and what they are capable of.

**Comment by one-forty** [comment]
Passed 500 lines! If a person is intelligent enough they will now be able to alert the others of dangers they have noticed, but if one of the others does not have enough focus they will not be aware of the danger.

**Comment by one-forty** [comment]
Ok I know it's late but I like this idea a lot more, I was running into a lot of bugs and brick walls with the previous idea to the point of frustration.

This will generate a battle log like Dwarf Fortress. It will generate a monster with randomized features and will have 4 warriors attempt to defeat it. I'm planning to add as much detail as possible.

I apologize if you were really looking forward to Diastora Forest but I feel like this will be more fun for both you and I.

**Comment by one-forty** [comment]
The user will be able to choose the variables they want to be used while generating the log, or they could choose it to be completely random, or a mix. This will hopefully lead to some entertaining experiments

**Comment by one-forty** [comment]
So... my pc restarted and apparently now the solution won't load. It says 'Attach' instead of 'Start'

I've tried everything I could find online to resolve this, if I can't fix it by tonight I'm going to close the issue and try again next year. It's too complex to rebuild in 9-10 days, especially as those days are very busy for me.

**Comment by one-forty** [comment]
The file for the applications actual design (don't know the term sorry) doesn't exist anymore. Something corrupted when it crashed?

I'm gonna try my best to remake it

**Comment by one-forty** [comment]
Here is an example I am proud of, this is just the very very start of the introduction, I have most of the actual code but I just haven't written the story yet, tell me if you would like another example. Enjoy :)

"This is a log recorded of an epic battle that took place between the feared KAYAROKK and three warriors, Cantila, Lypin, and Nymo...
The battle took place in the Xayawaan Castle while the cold snow was falling onto them...

KAYAROKK was a huge beast (around 10.12 times the size of a normal human), it had a fierce, blue, dragon body coated with spikes with a long tail and neck, it had large wings and the body looked muscular and  it had a demonic, horned head...
949682 people have fell victim to the dreaded KAYAROKK and its existence is rumored to be dated back to  dawn of the creation of the world..."

**Comment by LuRsT** [comment]
I say go for it @one-forty !

**Comment by hugovk** [comment]
> I made it way too advanced for what I had time left to do, I could post an unfinished version that still generates 50k words. Would you guys prefer that?

This is the unofficial motto of NaNoGenMo :) Go for it!
